lighthouse-core/audits/third-party-facades.js

```diff
@@ -0,0 +1,209 @@
+  description: 'Some third party embeds can be lazy loaded. ' +
+    'Consider replacing them with a facade until they are required. [Learn more](https://web.dev/efficiently-load-third-party-javascript/).',
+  /** Summary text for the result of a Lighthouse audit that identifies the third party code on a web page that can be lazy loaded with a facade alternative. This text summarizes the number of lazy loading facades that can be used on the page. Lazy loading means loading resources is deferred until they are needed. */
+  displayValue: `{itemCount, plural,
+  =1 {# facade alternative available}
+  other {# facade alternatives available}
+  }`,
+  /** Label for a table column that displays the name of the product that a URL is used for. A product is a piece of software used on the page. */
+  columnProduct: 'Product',
+  /**
+   * @description Template for a table entry that gives the name of a product which we categorize as video related.
+   * @example {YouTube Embedded Player} productName
+   */
+  categoryVideo: '{productName} (Video)',
+  /**
+   * @description Template for a table entry that gives the name of a product which we categorize as customer success related. Customer success means the product supports customers by offering chat and contact solutions.
+   * @example {Intercom Widget} productName
+   */
+  categoryCustomerSuccess: '{productName} (Customer Success)',
+  /**
+   * @description Template for a table entry that gives the name of a product which we categorize as marketing related.
+   * @example {Drift Live Chat} productName
+   */
+  categoryMarketing: '{productName} (Marketing)',
+  /**
+   * @description Template for a table entry that gives the name of a product which we categorize as social related.
+   * @example {Facebook Messenger Customer Chat} productName
+   */
+  categorySocial: '{productName} (Social)',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+/** @type {Record<string, string>} */
+const CATEGORY_UI_MAP = {
+  'video': UIStrings.categoryVideo,
+  'customer-success': UIStrings.categoryCustomerSuccess,
+  'marketing': UIStrings.categoryMarketing,
+  'social': UIStrings.categorySocial,
+};
+
+class ThirdPartyFacades extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'third-party-facades',
+      title: str_(UIStrings.title),
+      failureTitle: str_(UIStrings.failureTitle),
+      description: str_(UIStrings.description),
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'URL'],
+    };
+  }
+
+  /**
+   * @param {ThirdPartySummary.URLSummary[]} items
+   */
+  static condenseItems(items) {
+    items.sort((a, b) => b.transferSize - a.transferSize);
+    const splitIndex = items.findIndex((item) => item.transferSize < 1000);
```

Comment:
feels like we could roll up a little more, this index or a max of 6 or something similar to the other third party audit?